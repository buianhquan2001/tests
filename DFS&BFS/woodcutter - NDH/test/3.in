10
41 12
11 16
16 34
34 30
30 35
35 39
39 12
12 20
20 10
10 24
24 18
11 29
29 38
38 4
4 31
31 26
26 21
21 1
1 9
9 27
27 5
11 41
41 19
19 28
28 6
6 25
25 2
2 14
14 8
8 36
36 33
11 3
3 7
7 37
37 15
15 17
17 13
13 22
22 23
23 40
40 32
25 3
17 20
20 4
4 7
7 19
19 12
12 15
17 1
1 9
9 18
18 22
22 25
25 13
17 8
8 16
16 23
23 14
14 6
6 21
17 24
24 10
10 3
3 11
11 5
5 2
46 1
35 2
2 5
5 28
28 42
42 8
8 25
25 14
14 16
16 30
35 26
26 13
13 7
7 17
17 19
19 40
40 43
43 36
36 27
35 34
34 20
20 39
39 12
12 4
4 23
23 15
15 44
44 37
35 41
41 11
11 1
1 46
46 3
3 31
31 24
24 38
38 6
35 9
9 33
33 29
29 18
18 10
10 21
21 32
32 45
45 22
21 15
16 12
12 9
9 14
14 3
3 21
21 8
8 18
18 20
20 19
19 1
16 6
6 4
4 15
15 13
13 17
17 7
7 2
2 11
11 5
5 10
25 9
2 15
15 1
1 8
8 12
12 23
23 19
2 25
25 22
22 14
14 18
18 3
3 21
2 4
4 13
13 24
24 9
9 6
6 16
2 5
5 20
20 10
10 11
11 7
7 17
33 1
29 22
22 28
28 15
15 32
32 9
9 3
3 12
12 25
29 11
11 16
16 19
19 10
10 7
7 21
21 33
33 26
29 27
27 1
1 5
5 30
30 2
2 18
18 14
14 8
29 24
24 31
31 4
4 20
20 6
6 13
13 23
23 17
51 4
31 5
5 46
46 10
10 44
44 21
21 11
11 9
9 29
29 34
34 33
31 8
8 16
16 42
42 48
48 28
28 2
2 36
36 13
13 45
45 26
31 27
27 50
50 30
30 19
19 25
25 51
51 24
24 22
22 18
18 40
31 37
37 3
3 47
47 49
49 17
17 23
23 6
6 1
1 14
14 39
31 38
38 43
43 20
20 7
7 4
4 12
12 41
41 32
32 35
35 15
22 14
1 11
11 10
10 9
9 8
8 2
2 21
21 22
1 19
19 17
17 15
15 20
20 18
18 12
12 6
1 16
16 5
5 4
4 3
3 13
13 7
7 14
36 5
30 14
14 9
9 8
8 4
4 29
29 34
34 28
30 5
5 24
24 23
23 25
25 11
11 10
10 3
30 2
2 26
26 1
1 7
7 15
15 6
6 18
30 16
16 32
32 27
27 13
13 17
17 36
36 21
30 33
33 35
35 19
19 31
31 20
20 12
12 22
31 8
18 31
31 23
23 6
6 21
21 14
14 9
18 7
7 30
30 10
10 5
5 3
3 8
18 26
26 20
20 4
4 2
2 27
27 29
18 25
25 22
22 28
28 17
17 1
1 19
18 24
24 16
16 11
11 12
12 13
13 15
