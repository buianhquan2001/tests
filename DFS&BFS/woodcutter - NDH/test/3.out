0
13
41
0
0
29
31
0
11
0
